src/executables/csql.c

```diff
@@ -1076,7 +1086,17 @@ csql_do_session_cmd (char *line_read, CSQL_ARGUMENT * csql_arg)
       break;
 
     case S_CMD_TRIGGER:
-      csql_help_trigger ((argument[0] == '\0') ? NULL : argument);
+      if (argument[0] == '\0')
+	{
+	  csql_help_trigger (NULL);
+	}
+      else
+	{
+	  char realname[SM_MAX_IDENTIFIER_LENGTH] = { '\0' };
+	  sm_user_specified_name (argument, realname, SM_MAX_IDENTIFIER_LENGTH);
```

Comment:
Isn't it necessary for the S_CMD_INFO case as well?
```
info stat table
```

Reply:
Below, the stats_dump() function is called and the class-name is found in the sm_find_class() function.
The sm_find_class() function calls the sm_user_specified_name() function to find the class-name.
```
;info stat class-name
```